keyboards/lazydesigners/dimple/ortho/keymaps/via/keymap.c

```diff
@@ -0,0 +1,53 @@
+/* Copyright 2021 LAZYDESIGNERS
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+#define LT2_TAB LT(2, KC_TAB)
+#define LT1_SPC LT(1, KC_SPC)
+
+/*
+ * ,------------------------------------------------------------------.
+ * |Esc    | Q  | W  | E  | R  | T  | Y  | U  | I  | O  | P  |  Bspc  |
+ * |------------------------------------------------------------------|
+ * |Tab/LT2| A  | S  | D  | F  | G  | H  | J  | K  | L  | '  |  |Enter|
+ * |------------------------------------------------------------------|
+ * | Shift | Z  | X  | C  | V  | B  | N  | M  | ,  | .  | Up | MO(2)  |
+ * |------------------------------------------------------------------|
+ *      |Ctrl|Gui |Alt | Spc/LT1  | MO(3)|   Spc   |Left|Down|Rght|
+ *      `---------------------------------------------------------'
+ */
+
+ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
```

Comment:
This should have 4 layers

Reply:
Sorry about that. 'MO(3)' was a mistake.

Reply:
Regardless, this still needs to have 4 layers. 

By default, VIA assumes 4 layers, and if they're not all populated, then the unpopulated layers will be filled with random garbage.  

So you need to either have 4 layers, or you need to set the define that limits it to 3 layers.

Reply:
I see. Thank you!